overlord/snapstate/conflict.go

```diff
@@ -176,18 +197,7 @@ func CheckChangeConflictMany(st *state.State, instanceNames []string, ignoreChan
 
 	for _, task := range st.Tasks() {
 		chg := task.Change()
-		if chg == nil || chg.Status().Ready() {
-			continue
-		}
-		if ignoreChangeID != "" && chg.ID() == ignoreChangeID {
-			continue
-		}
-		if chg.Kind() == "become-operational" {
```

Comment:
I merged a change adding a new kind to this condition so it needs to be included in `isIrrelevantChange`

Reply:
I've rebase to master and resolved the conflict, thanks for the review